### ISSUE TITLE
Parameters for training on the small dataset on GPU

### ISSUE THREAD

**Issue by ghalib2021** [facebookresearch/swav#78]
At first i want to take this oppourtunity to thanks to the authors for this work. I have runned the code and few experiments using SWAV. I tried to retrain the model on the small dataset which have 1300 images. following is parameters settings. My accuracy is quite low even lower than using pretrained model for feature extraction. Following are my parameters

python -m torch.distributed.launch --nproc_per_node=1 main_swav.py \
--data_path /content/datasets/train/ \
--epochs 400 \
--base_lr 0.6 \
--final_lr 0.0006 \
--warmup_epochs 0 \
--batch_size 64 \
--size_crops 224 96 \
--nmb_crops 2 6 \
--min_scale_crops 0.14 0.05 \
--max_scale_crops 1. 0.14 \
--use_fp16 true \
--freeze_prototypes_niters 5005 \
--queue_length 3840 \
--epoch_queue_starts 15

**Please help in suggesting which paramters i was wrong due to which i was unable to achieve performance. The total number  of classes in the orifinal dataaset is 15**

**Comment by xgyyao** [comment]
hi @ghalib2021 , is the size of model you trained the same as pretrained model?
the size of swav_800ep_pretrain.pth.tar is 109M
but the size of my trained model is 217M
